sdcm/kcl_thread.py

```diff
@@ -0,0 +1,80 @@
+import os
+import random
+import logging
+import uuid
+
+from sdcm.stress_thread import format_stress_cmd_error, DockerBasedStressThread
+from sdcm.utils.docker_utils import RemoteDocker
+from sdcm.sct_events import StressEvent, Severity
+
+LOGGER = logging.getLogger(__name__)
+
+
+class KclStressEvent(StressEvent):
+    pass
+
+
+class KclStressThread(DockerBasedStressThread):  # pylint: disable=too-many-instance-attributes
+
+    def run(self):
+        _self = super().run()
+        # wait for the KCL thread to create the tables, so the YCSB thread beat this one, and start failing
+        time.sleep(120)
+        return _self
+
+    def build_stress_cmd(self):
+        if self.params.get('alternator_use_dns_routing'):
+            target_address = 'alternator'
+        else:
+            if getattr(self.node_list[0], 'parent_cluster'):
+                target_address = self.node_list[0].parent_cluster.get_node().ip_address
+            else:
+                target_address = self.node_list[0].ip_address
+        stress_cmd = f"./gradlew run --args=\' {self.stress_cmd.replace('hydra-kcl', '')} -e http://{target_address}:{self.params.get('alternator_port')} \'"
+        return stress_cmd
+
+    def _run_stress(self, loader, loader_idx, cpu_idx):
+        dns_options = ""
+        if self.params.get('alternator_use_dns_routing'):
+            dns = RemoteDocker(loader, "scylladb/hydra-loaders:alternator-dns-0.2",
```

Comment:
shouldn't you call `run` on this somewhere?

Reply:
I think RemoteDocker in this case, is starting that image, the run command is need when you have extra command to run
i.e. we run the image with pipe to /dev/null, and then do `docker exec` to run the stress comamnd

this little thing, it's our poorsman loadbalancer base on DNS, it's run in the background, parallel to the actual stress command